File: backend/Models/Submission.cs
namespace backend.Models
{
    public class Submission
    {
        public int Id { get; set; }
        public int AssignmentId { get; set; }
        public int StudentId { get; set; }
        public string FilePath { get; set; } = string.Empty;
        public DateTime SubmittedAt { get; set; } = DateTime.UtcNow;
        public string Status { get; set; } = "Pending"; // Pending, Graded
        public int? Grade { get; set; }
    }
}

File: frontend/src/Types/Submission.ts
export interface Submission {
  id: number;
  assignmentId: number;
  studentId: number;
  filePath: string;
  submittedAt: string;
  status: string;
  grade?: number;
}
export interface StudentSubmission {
  id: number;
  studentId: number;
  assignmentId: number;
  fileUrl: string;
  dateSubmitted: string;
  grade?: string;
  filePath: string;
  status: string;
    submittedAt: string;
}

